torch/_torch_docs.py

```diff
@@ -559,13 +559,7 @@ def merge_dicts(*dicts):
 :math:`(n \times m)`.
 
 For inputs of type `FloatTensor` or `DoubleTensor`, arguments :attr:`beta` and
-:attr:`alpha` must be real numbers, otherwise they should be integers
-
-.. warning::
-    This function is deprecated and may be removed in a future release.
-    It can be implemented using :func:`torch.outer` as
-    ``alpha * torch.outer(vec1, vec2) + beta * input`` when :attr:`beta` is not zero,
-    and as ``alpha * torch.outer(vec1, vec2)`` when :attr:`beta` is zero.
+:attr:`alpha` must be real numbers, otherwise they should be integers.
```

Comment:
This paragraph is out of date and the term "real numbers" is ambiguous.

Reply:
Thanks so much for the suggestion, the paragraph has been removed.